docs/getting-started.md

```diff
@@ -0,0 +1,184 @@
+Labels:                 app=httpd
+Annotations:            kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"v1","kind":"Service","metadata":{"annotations":{},"creationTimestamp":null,"labels":{"app":"httpd"},"name":"httpd","namespace":"default"...
+Selector:               app=httpd
+Type:                   LoadBalancer
+IP:                     10.0.0.249
+Port:                   httpd-8080      8080/TCP
+NodePort:               httpd-8080      30692/TCP
+Endpoints:              172.17.0.4:80
+Session Affinity:       None
+Events:                 <none>
+```
+
+## Minishift and Kedge
+
+In this guide, we'll deploy an OpenShift-compatible Kedge file to an OpenShift cluster.
+
+Requirements:
+  - [minishift](https://github.com/minishift/minishift)
+  - [kedge](https://github.com/kedgeproject/kedge)
+  - An OpenShift route created
+
+__Note:__ The service will NOT be accessible until you create an OpenShift route with `oc expose`. You must also have a virtualization environment setup. By default, `minishift` uses KVM.
+
+__Start `minishift`:__
+
+[Minishift](https://github.com/minishift/minishift) is a tool that helps run OpenShift locally using a single-node cluster inside of a VM. Similar to [minikube](https://github.com/kubernetes/minikube).
+
+```sh
+$ minishift start
+Starting local OpenShift cluster using 'kvm' hypervisor...
+-- Checking OpenShift client ... OK
+-- Checking Docker client ... OK
+-- Checking Docker version ... OK
+-- Checking for existing OpenShift container ... OK
+...
+```
+
+__Login as developer within `minishift`:__
+
+Login as developer if you haven't done so already within `minishift`.
+
+
+```sh
+▶ oc login -u developer          
+Server [https://localhost:8443]: https://192.168.42.113:8443
+The server uses a certificate signed by an unknown authority.
+You can bypass the certificate check, but any data you send to the server could be intercepted by others.
+Use insecure connections? (y/n): y 
+
+Authentication required for https://192.168.42.113:8443 (openshift)
+Username: developer
+Password: 
+Login successful.
+
+You have one project on this server: "myproject"
+
+Using project "myproject".
+Welcome! See 'oc help' to get started.
+```
+
+__Download the [Guestbook Demo files](https://github.com/kedgeproject/kedge/tree/master/examples/guestbook-demo) from GitHub:__
+
+Due to OpenShift using non-root containers, we will be using an OpenShift-compatible demo. In particular, the highly-used "Guestbook" Kubernetes demo.
+
+```sh
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/guestbook-demo/backend.yaml
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/guestbook-demo/frontend.yaml
+wget https://raw.githubusercontent.com/kedgeproject/kedge/master/examples/guestbook-demo/db.yaml
+```
+
+__Deploy directly to your OpenShift cluster with `kedge apply`:__
+
+Run `kedge apply -f backend.yaml -f frontend.yaml -f db.yaml` in the same directory as your example files.
```

Comment:
If we have only kedge files in the directory then we can always do `kedge apply -f .`, `.` being the directory all files in that directory will be taken.

Reply:
IMO, I think we should keep the same.